tests/ResponseHeadersTest.php

```diff
@@ -0,0 +1,271 @@
+<?php
+
+namespace Jasny\HttpMessage;
+
+use Jasny\HttpMessage\Tests\AssertLastError;
+use PHPUnit_Framework_TestCase;
+
+/**
+ * @covers Jasny\HttpMessage\ResponseHeaders
+ * @runTestsInSeparateProcesses
+ */
+class ResponseHeadersTest extends PHPUnit_Framework_TestCase
+{
+    use AssertLastError;
+    
+    /**
+     * @var ResponseHeaders
+     */
+    protected $headers;
+
+    public function setUp()
+    {
+        $this->headers = new ResponseHeaders();
+    }
+
```

Comment:
Unset `$_GLOBALS['headersList']` and `$_GLOBALS['responseCode']` after each test.